.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -1,74 +1,75 @@
-name: Have you encountered an issue?
-description: Report an issue with Winston.
-title: "[Bug]: "
-labels: ["Bug", "Needs Investigation"]
-body:
-  - type: markdown # Form Header
-    attributes:
-      value: >-
-        This issue form is for reporting bugs only!
-  - type: input # Search Terms
-    validations:
-      required: true
-    attributes:
-      label: 🔎 Search Terms
-      description: >-
-        What search terms did you use when trying to find an existing bug report, looking in both open and closed issues?
-        List them here so people in the future can find this one more easily.
-  - type: textarea # Problem Definition
-    validations:
-      required: true
-    attributes:
-      label: The problem
-      description: >-
-        Please provide a clear and concise description of the problem you've encountered and what the your
-        expected behavior was.
-  - type: markdown # Environment Section Header
-    attributes:
-      value: |
-        ## Environment
-  - type: input # Affected Version Input
-    id: winston-version
-    validations:
-      required: true
-    attributes:
-      label: What version of Winston presents the issue?
-      placeholder: v3.4.0
-      description: >
-        Can be found by running one of the following (depending on your package manager of choice):
-        - `npm list winston`
-        - `yarn list --pattern winston`
-  - type: input # Affected Version Input
-    id: node-version
-    validations:
-      required: true
-    attributes:
-      label: What version of Node are you using?
-      placeholder: v16.8.0
-      description: >
-        Can be found by running the following: `node -v`
-  - type: input # Last Known Working Version
-    attributes:
-      label: If this worked in a previous version of Winston, which was it?
-      placeholder: v3.0.0
-      description: >
-        If known, otherwise please leave blank.
-  - type: markdown # Details Section Header
-    attributes:
-      value: |
-        # Details
-  - type: textarea # Minimum Working Example Input
-    attributes:
-      label: Minimum Working Example
-      description: |
-        If you can, providing an MWE to reproduce the issue you're encountering can greatly speed up
-        investigation into the issue by one of our maintainers, or anyone else in the community who's looking 
-        to get involved.
-        
-        This can be as simple as a script, a link to a repo, etc.
-      render: yaml
-  - type: textarea # Additional Information
-    attributes:
-      label: Additional information
-      description: >
-        If you have any additional information for us that you feel will be valuable, please use the field below.
+name: Have you encountered an issue?

```

Comment:
Not sure why the diffs are against the entire file. This may have to due with the `.editorconfig` configuration for linebreaks? Will have to inspect these files to see if line-endings changed or not. In the meantime this can be more easily reviewed by enabling the `hide whitespace` option for the diff itself.

Reply:
Verified that line-endings were changed from previous. This is almost guaranteed to be because of the editor config. As such I'm removing that configuration and will let systems/git handle however it sees fit
<img width="601" alt="Screen Shot 2022-02-12 at 2 26 41 PM" src="https://user-images.githubusercontent.com/10217028/153729039-a4e052f5-74b5-44f4-a28a-7d975fff6e8c.png">
.